### PR TITLE
Add clickable Links/URL

### PR DESCRIPTION
**Pull request by chloe-codes1**
## Tests

I added the following tests:

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

[wehack2021]-[Cloud]

### CODE REVIEW

**Comment by 0916dhkim** [comment]
@chloe-codes1  This looks promising! I will review/approve later today :+1:

File: client/src/components/uis/MessageListItem.tsx [hunk 1]
```diff
@@ -163,6 +171,10 @@ const decorateDate = (createdAt: string): string => {
   return date.fromNow();
 };
 
+const handleUrlPress = (url: string): void => {
+  Linking.openURL(url);
```
Comment (0916dhkim) [inline client/src/components/uis/MessageListItem.tsx:175]: I was not sure if this would work on web, but it does 👍

**Review by 0916dhkim — APPROVED** [review]
Looks good to me 💯 
There is one thing I noticed when testing this feature: "https://google.com" is parsed correctly while "google.com" (without protocol) is not, but I think it is a minor issue we can live with.

I think this is good to be merged after fixing hardcoded color.

File: client/src/components/uis/MessageListItem.tsx [hunk 2]
```diff
@@ -113,6 +114,13 @@ const StyledMyMessage = styled.View`
   border-radius: 3px;
 `;
 
+const styles = StyleSheet.create({
+  url: {
+    color: '#1E6EFA',
```
[the remark this answers is not in the document]
Comment (chloe-codes1) [reply client/src/components/uis/MessageListItem.tsx]: Thank you for your suggestion!
I have fixed hardcoded color as you mentioned above.

File: client/src/components/uis/MessageListItem.tsx (hunk 1 above)
Comment (geoseong) [reply client/src/components/uis/MessageListItem.tsx:176]: I checked this works on web(in Chrome)!

**Review by geoseong — APPROVED** [review]
LGTM!
the package [react-native-parsed-text](https://github.com/taskrabbit/react-native-parsed-text) looks cool and easy to use.

File: client/src/components/uis/MessageListItem.tsx [hunk 3]
```diff
@@ -113,6 +115,13 @@ const StyledMyMessage = styled.View`
   border-radius: 3px;
 `;
 
+const styles = StyleSheet.create({
+  url: {
+    color: light.link,
```
Comment (hyochan) [inline client/src/components/uis/MessageListItem.tsx:120]: Why not `${({ theme }) => theme.link}` here?

**Review by hyochan — APPROVED** [review]
Thanks for your contribution :100:

File: client/src/components/uis/MessageListItem.tsx (hunk 3 above)
Comment (chloe-codes1) [reply client/src/components/uis/MessageListItem.tsx]: @hyochan 
I’ve made a few adjustment to StyleSheet.
Thanks for your suggestion!

**Comment by hyochan** [comment]
@all-contributors Please add @chloe-codes1 for code

**Pull request merged**